ci/deploy/docs.sh

```diff
@@ -10,10 +10,16 @@ pushd docs/book/
 # for now depending on global project devshell 
 mdbook build # build output is in the `book` directory
 
-# TODO: deploy book to codetracer.com ? ask Zahary?
-
-echo '###############################################################################'
-echo "TODO upload book"
-echo '###############################################################################'
+# If the gh-pages branch already exists, this will overwrite it
+# so that the history is not kept, which can be very expensive.
+git worktree add --orphan -B gh-pages gh-pages
```

Comment:
i've never used worktrees, interesting

Reply:
I just googled and copied what it gave me